src/app/services/api.service.ts

```diff
@@ -60,4 +60,53 @@ export class ApiService {
       },
     });
   }
+
+  public userCreate (name: string, firstname: string, lastname: string, organizationId: number) {
+    const typeId = this.settingsService.typeId('users');
```

Comment:
It could be better to rename the function `getTypeId` or perhaps `getTypeIdByInternalname`

Reply:
It is done. I also renamed the `Item.propValue` in `Item.getPropValueByInternalname`